spectool/specdata.xml

```diff
@@ -38,7 +38,7 @@
   <element name="TimecodeScale" level="2" id="0x2AD7B1" type="uinteger" mandatory="1" minver="1" default="1000000">Timestamp scale in nanoseconds (1.000.000 means all timestamps in the segment are expressed in milliseconds).</element>
   <!-- <element name="TimecodeScaleDenominator" level="2" id="0x2AD7B2" type="uinteger" mandatory="1" minver="4" default="1000000000">Timestamp scale numerator, see <a href="http://www.matroska.org/technical/specs/index.html#TimecodeScale">TimecodeScale</a>.</element>
   TimecodeScale When combined with <a href="http://www.matroska.org/technical/specs/index.html#TimecodeScaleDenominator">TimecodeScaleDenominator</a> the Timestamp scale is given by the fraction TimecodeScale/TimecodeScaleDenominator in seconds.-->
-  <element name="Duration" level="2" id="0x4489" type="float" minver="1" range="&gt; 0">Duration of the segment (based on TimecodeScale).</element>
+  <element name="Duration" level="2" id="0x4489" type="float" minver="1" range="&gt;0x0p+1">Duration of the segment (based on TimecodeScale).</element>
```

Comment:
needs spacing after &gt; for clarity. Also 0x0p+0 is a clearer representation of 0.0